pubsub/topic.go

```diff
@@ -200,6 +198,33 @@ type TopicConfig struct {
 	// Schema defines the schema settings upon topic creation. This cannot
 	// be modified after a topic has been created.
 	SchemaSettings *SchemaSettings
+
+	// RetentionDuration configures the minimum duration to retain a message
+	// after it is published to the topic. If this field is set, messages published
+	// to the topic in the last `message_retention_duration` are always available to subscribers.
```

Comment:
Does it make sense to s/message_retention_duration/RetentionDuration in this comment?